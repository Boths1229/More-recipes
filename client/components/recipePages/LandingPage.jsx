import React from 'react';
import IndexHeader from '../common/IndexHeader';
import Carousel from '../Include/carousel';
import IndexMain from '../Include/IndexMain';
import IndexFooter from '../common/IndexFooter';

/**
 * @description functional component to render the landing page
 *
 * @returns {JSX} return JSX
 */
const LandingPage = () => (
  <div>
    <IndexHeader />
    <Carousel />
    <IndexMain />
    <IndexFooter />
  </div>
);


export default LandingPage;
