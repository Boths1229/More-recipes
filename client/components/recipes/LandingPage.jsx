import React from 'react';
import IndexHeader from '../common/IndexHeader';
import Carousel from '../Include/carousel';
import IndexMain from '../Include/IndexMain';
import IndexFooter from '../common/IndexFooter';

const LandingPage = () => {
  return (
    <div>
      <IndexHeader />
      <Carousel />
      <IndexMain />
      <IndexFooter />
    </div>
  );
};

export default LandingPage;
