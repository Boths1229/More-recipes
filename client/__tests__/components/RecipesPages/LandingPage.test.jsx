import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from '../../../components/recipePages/LandingPage';

describe('Landing page component', () => {
  it('matches snapshot', () => {
    const output = shallow(<LandingPage />);
    expect(output).toMatchSnapshot();
  });
});
