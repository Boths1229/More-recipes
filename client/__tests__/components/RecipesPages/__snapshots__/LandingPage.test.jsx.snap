// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing page component matches snapshot 1`] = `
<div>
  <IndexHeader />
  <Carousel />
  <IndexMain />
  <IndexFooter />
</div>
`;
